### ISSUE TITLE
Yocto qt blog post from http://blog.unixshell.se/?p=27 unavailable

### ISSUE THREAD

**Issue by MaxMagazin** [erikboto/meta-erborpi#3]
Hello Erik,

Well, this issue is not directly connected with this github repo (sorry about that), but rather with your blog. I already mentioned the problem in #1, but as the issue is closed, I was not sure your received any notifications about it. I also understood you restarted your blog, but is there any chance you restore somehow your post about yocto with qt and publish it again?

Best regards,
Maxim.

**Comment by erikboto** [comment]
Hi,

Sorry for not replying earlier.

The blog was kind of restarted after I moved to another server. I did manage to put the old blog back up under another domain temporarily, have a look at: http://blog.bendoverconsulting.se/?p=27. I would guess most things are handled quite well using only meta-raspberrypi nowadays, so keep that in mind when reading the (quite old) blog post. The last release I updated stuff for was rocko, see https://github.com/erikboto/pi-manifest/tree/rocko

I'll see if I can get some time to update to latest yocto release, and maybe make a new blog post with more current information.

Good luck!
Erik
